### ISSUE TITLE
translatable field causes error

### ISSUE THREAD

**Issue by rallek** [Guite/MostGenerator#721]
````
Fatal error: Class 'Gedmo\Translatable\Entity\AbstractTranslation' not found in /home/www/141.koester123.de/Zikula_Core-1.4.1/modules/VerySimplePages/lib/VerySimplePages/Entity/Base/PageTranslation.php on line 22

Fatal error: Uncaught exception 'UnexpectedValueException' with message 'RecursiveDirectoryIterator::__construct(//var/lib/polkit-1): failed to open dir: Permission denied' in /home/www/141.koester123.de/Zikula_Core-1.4.1/vendor/symfony/symfony/src/Symfony/Component/Debug/FatalErrorHandler/ClassNotFoundFatalErrorHandler.php:148 Stack trace: #0 [internal function]: RecursiveDirectoryIterator->__construct('//var/lib/polki...', 4096) #1 /home/www/141.koester123.de/Zikula_Core-1.4.1/vendor/symfony/symfony/src/Symfony/Component/Debug/FatalErrorHandler/ClassNotFoundFatalErrorHandler.php(148): RecursiveDirectoryIterator->getChildren() #2 /home/www/141.koester123.de/Zikula_Core-1.4.1/vendor/symfony/symfony/src/Symfony/Component/Debug/FatalErrorHandler/ClassNotFoundFatalErrorHandler.php(117): Symfony\Component\Debug\FatalErrorHandler\ClassNotFoundFatalErrorHandler->findClassInPath(false, 'AbstractTransla...', 'ZikulaSecurityC...') #3 /home/www/141.koester123.de/Zikula_Core-1.4.1/vendor/symfony/symfony/src/Symfony/Component/Debug/F in /home/www/141.koester123.de/Zikula_Core-1.4.1/vendor/symfony/symfony/src/Symfony/Component/Debug/FatalErrorHandler/ClassNotFoundFatalErrorHandler.php on line 148
````
I used the webgen from 26.12.2015 within zk1.4.1.
The model is generated zk136

**Comment by cmfcmf** [comment]
Looks like a permission issue? 
>  failed to open dir: Permission denied

**Comment by rallek** [comment]
it should install like any other module I guess. There should not be any permission issue.

**Comment by Guite** [comment]
Fixed in b6f62fa5efc6a9d27f27ffa6ecfd5d401b2d158f
